transcom/mymove
# B 20978 int show links for documents for DDASIT and DDDSIT

## [B-20978](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1001296)

## Summary

This backlog adds links to download documents uploaded by the Prime for DDASIT and DDDSIT service items. Previously the links only displayed for Origin SIT service items. 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Prime Simulator
2. You can use any shipment and add a service item for destination SIT or use the DSTSIT move code which already has SIT service items added.
3. Click the Upload Document button for DDASIT and upload a document then save. Note the name of the document so you can verify the proper file link is displayed.
4. Click the Upload Document button for DDDSIT and upload a document.
5. Logout and login as a TOO and do a search for DSTSIT under move search or find the shipment you used and select it.
6. Go to the Move Task Order page and scroll down and verify that there are links for the uploaded documents in the service items section for DDASIT and DDDSIT.
![image](https://github.com/user-attachments/assets/713b06c4-925b-475f-bf1b-5648107a9fe4)


### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
